staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -367,6 +367,7 @@ type ListOptions struct {
 	ResourceVersion string `json:"resourceVersion,omitempty" protobuf:"bytes,4,opt,name=resourceVersion"`
 	// Timeout for the list/watch call.
 	// This limits the duration of the call, regardless of any activity or inactivity.
+	// This timeout don't change the client timeout.
```

Comment:
How about just modify line 369 to "This limits the duration of the call on the server-side"